base/compiler/ssair/inlining.jl

```diff
@@ -630,14 +630,7 @@ function ir_inline_unionsplit!(compact::IncrementalCompact, idx::Int,
     end
     bb += 1
     # We're now in the fall through block, decide what to do
-    if fully_covered
-        if !params.trust_inference
-            e = Expr(:call, GlobalRef(Core, :throw), FATAL_TYPE_BOUND_ERROR)
```

Comment:
Any reason you didn't remove the `FATAL_TYPE_BOUND_ERROR` definition?

Reply:
was this the only use of it? If so the reason is because I didn't notice that there weren't any other uses.

Reply:
https://github.com/JuliaLang/julia/pull/50465